### PR TITLE
Fix several typos in the docs

### PR DESCRIPTION
**Pull request by deepio**
Nice project, looking forward to what comes next. Keep up the great work.

### CODE REVIEW

File: docs/0.4.x/en-US/what-is-perseus.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Alright, you're pretty familiar with what web development is, and why we tend to
 
 Put simply, JS is [a bit of a mess](https://medium.com/netscape/javascript-is-kinda-shit-im-sorry-2e973e36fec4). It's dynamically-typed, and executed at runtime, meaning you can't really catch bugs while you're coding. Sure, an IDE helps with this by showing you squiggly red lines, but it still won't stop you from forgetting about passing a certain argument to a function. TypeScript helps with this by introducing stricter typing rules, but it's really an addition on top of already existing JavaScript, and, let's be honest, how many times have you had to search up solutions for getting your `tsconfig` to work?
 
-[Rust](https://rust-lang.org), on the other hand, is generally thought of as a systems programming language, meaning it's much lower-level and closer to the hardware, letting you do things like memory management more manually. It's certainly got a much steeper learning curve, but, let's walk through a quick example. Imagine you have a variable `data` that contains a very large amount of information. Obviously, copying this is going to slow your program down, so we want to avoid that if possible. In JS, you could do something like this:
```
Comment (arctic-hen7) [inline docs/0.4.x/en-US/what-is-perseus.md:31]: The phrase "a variable `data`" seems fine to me...

File: docs/next/en-US/what-is-perseus.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Alright, you're pretty familiar with what web development is, and why we tend to
 
 Put simply, JS is [a bit of a mess](https://medium.com/netscape/javascript-is-kinda-shit-im-sorry-2e973e36fec4). It's dynamically-typed, and executed at runtime, meaning you can't really catch bugs while you're coding. Sure, an IDE helps with this by showing you squiggly red lines, but it still won't stop you from forgetting about passing a certain argument to a function. TypeScript helps with this by introducing stricter typing rules, but it's really an addition on top of already existing JavaScript, and, let's be honest, how many times have you had to search up solutions for getting your `tsconfig` to work?
 
-[Rust](https://rust-lang.org), on the other hand, is generally thought of as a systems programming language, meaning it's much lower-level and closer to the hardware, letting you do things like memory management more manually. It's certainly got a much steeper learning curve, but, let's walk through a quick example. Imagine you have a variable `data` that contains a very large amount of information. Obviously, copying this is going to slow your program down, so we want to avoid that if possible. In JS, you could do something like this:
```
Comment (arctic-hen7) [inline docs/next/en-US/what-is-perseus.md:31]: Same thing here.

**Review by arctic-hen7 — CHANGES_REQUESTED** [review]
Thank you very much! Just one minor thing and this should be good to merge!

File: docs/next/en-US/what-is-perseus.md (hunk 2 above)
Comment (deepio) [reply docs/next/en-US/what-is-perseus.md]: done

File: docs/0.4.x/en-US/what-is-perseus.md (hunk 1 above)
Comment (deepio) [reply docs/0.4.x/en-US/what-is-perseus.md]: done

**Review by arctic-hen7 — APPROVED** [review]
Great, much appreciated! (Sorry for the slow review.)

**Pull request merged**